### PR TITLE
Orbis crazy boogie

### PR DESCRIPTION
**Pull request by OrbisAnima**
Added the custom items that were under my wing:

-Tassald's Energy Luger ( /obj/item/weapon/gun/energy/gun/tluger )
   Special note: the kill setting of both this and the Laser Luger don't do any damage, this is becouse the beam they use no longer does any damage in this code. Might be a good idea to change the beam both weapons use.
-Scree's Pompom ( /obj/item/clothing/head/fluff/pompom )
-Richard's Holochain. ( /obj/item/weapon/melee/fluff/holochain )

Additionally, added two fluff accesories to the demi machine, succubus dark and succubus crimson, both are different from the flappy bat wings since they have tails.

Couldn't find the shrink ray, so Aronai's request of a home made shrinkray is still pending.

### CODE REVIEW

**Comment by JoanRisu** [comment]
I'll have to upload a quick patch that adds unique lasers for the Lugers then.

**Comment by OrbisAnima** [comment]
with a new beam we might be able to fix it, even a modified emiter beam with the damage you consider appropiate. Since that's a balance thing I didn't want to meddle much into it

**Comment by OrbisAnima** [comment]
I won't be here until tomorrow, so I won't be able to answer for this pull as I planned originally. I followed Joan's line and placed the luger in the custom guns file, Wish you all the luck, and I hope I can be for the big event tomorrow.

File: code/modules/customitems/custom_guns.dm [hunk 1]
```diff
@@ -58,6 +58,36 @@
...
+	modifystate = "tlugerstun"
+
+/obj/item/weapon/gun/energy/gun/tluger/attack_self(mob/living/user as mob)
+	switch(mode)
+		if(0)
+			mode = 1
+			charge_cost = 100
+			fire_sound = 'sound/weapons/eluger.ogg'
+			user << "<span class='warning'>[src.name] is now set to kill.</span>"
+			projectile_type = /obj/item/projectile/beam/emitter
```
Comment (JoanRisu) [inline code/modules/customitems/custom_guns.dm]: Replace /obj/item/projectile/beam/emitter with /obj/item/projectile/beam/eluger for projectile_type and I'll do the rest.

**Comment by OrbisAnima** [comment]
There we go, all done Joan!

Ace, this is ready to be pulled, just notice that Joan's beam PR also needs to be pulled for it to work

**Comment by OrbisAnima** [comment]
I will remake this PR, since the guns.dmi file was modified.

**Comment by JoanRisu** [comment]
Perhaps syncing this branch with the latest merge should help resolve some stuff

**Comment by SpadesNeil** [comment]
This branch has conflicts that must be resolved.

It's because I merged Joan's eluger fix.

**Comment by OrbisAnima** [comment]
I'll remake it, don't worry

**Pull request closed without merging**